### PR TITLE
Drop captureHandle() from the event

### LINKED ISSUE #50: CaptureHandleChangeEvent should just be Event
The [doc](https://w3c.github.io/mediacapture-handle/identity/index.html#capturing-side) says _"Capturing applications who are permitted to [observe](https://w3c.github.io/mediacapture-handle/identity/index.html#dfn-observable) a track's [CaptureHandle](https://w3c.github.io/mediacapture-handle/identity/index.html#dom-capturehandle) have two ways of reading it."_ 

This is one too many. The second way:
```js
let handle;
track.oncapturehandlechange = event => handle = event.captureHandle();
```
...seems entirely redundant, since this should suffice:
```js
let handle;
track.oncapturehandlechange = ({target}) => handle = target.getCaptureHandle();
```
I believe the common pattern in situations like this is to just fire a vanilla `Event`.

### PR DESCRIPTION
**Pull request by eladalon1983**
Fixes #50.

### CODE REVIEW

**Review by jan-ivar — CHANGES_REQUESTED** [review]

File: identity/index.html [hunk 1]
```diff
@@ -225,9 +225,7 @@ <h3>MediaDevices.setCaptureHandleConfig()</h3>
               <li>
                 For every capturing application for which the new [=observable=] {{CaptureHandle}}
                 is different than prior to the call to {{MediaDevices/setCaptureHandleConfig}}, an
-                event of type {{CaptureHandleChangeEvent}} must be fired with
-                {{CaptureHandleChangeEvent/captureHandle}} set to the new [=observable=]
-                {{CaptureHandle}}.
+                event named "capturehandlechange" must be fired.
```
Comment (beaufortfrancois) [inline identity/index.html:228]: I believe it should say

```
the user agent MUST queue a task to fire an event with the name "capturehandlechange" at track.
```

Comment (beaufortfrancois) [reply identity/index.html:228]: Track needs to be defined there as well

File: identity/index.html [hunk 2]
```diff
@@ -370,42 +368,9 @@ <h4><dfn>CaptureHandleChangeEvent</dfn></h4>
...
-            <dt>
-              <dfn>captureHandle</dfn>
-            </dt>
-            <dd>
-              <p>The track's {{CaptureHandle}} at the time the event was fired.</p>
-            </dd>
-          </dl>
         </section>
         <section id="oncapturehandlechange">
           <h4><dfn>oncapturehandlechange</dfn></h4>
```
Comment (beaufortfrancois) [inline identity/index.html:376]: `oncapturehandlechange` below should be defined as `Fired on a MediaStreamTrack when its observable CaptureHandle changes`.

**Review by beaufortfrancois — CHANGES_REQUESTED** [review]

**Comment by jan-ivar** [comment]
Editor's can integrate once all CaptureHandleChangeEvent WebIDL is removed.

File: identity/index.html (hunk 1 above)
Comment (jan-ivar) [reply identity/index.html:228]: This will be fixed in #56.

Comment (jan-ivar) [reply identity/index.html:228]: Should be fixed by #56.

File: identity/index.html (hunk 2 above)
Comment (jan-ivar) [reply identity/index.html:376]: Should be fixed by #56.

File: identity/index.html (hunk 1 above)
Comment (eladalon1983) [reply identity/index.html:228]: To keep things more readable, I will switch from the passive voice ("an event must be fired") to the active voice ("the user agent MUST fire an event").

**Pull request merged**